src/translations/en.json

```
@@ -2033,26 +2040,33 @@
               "geo_location_sources": "Geolocation Sources",
               "dark_mode": "Dark Mode?",
               "default_zoom": "Default Zoom",
-              "source": "Source"
+              "source": "Source",
+              "description": "A card that allows you to display entities on a map."
             },
             "markdown": {
               "name": "Markdown",
-              "content": "Content"
+              "content": "Content",
+              "description": "Markdown card is used to render Markdown."
             },
             "media-control": {
-              "name": "Media Control"
+              "name": "Media Control",
+              "description": "The media control card is used to display Media Player entities on an interface with easy to use controls."
             },
             "picture": {
-              "name": "Picture"
+              "name": "Picture",
+              "description": "A very simple card that allows you to set an image to use for navigation to various paths in your interface or to call a service."
             },
             "picture-elements": {
-              "name": "Picture Elements"
+              "name": "Picture Elements",
+              "description": "Picture elements card is one of the most versatile types of cards. The cards allow you to position icons or text and even services! On an image based on coordinates."
             },
             "picture-entity": {
-              "name": "Picture Entity"
+              "name": "Picture Entity",
+              "description": "Displays the entity in form of an image. Instead of images from URL it can also show the picture of camera entities."
             },
             "picture-glance": {
-              "name": "Picture Glance"
+              "name": "Picture Glance",
+              "description": "Show an image card and corresponding entity states as icon. The entities on the right side allow toggle actions, others show the more-info-dialog."
```

Should we write `more-info-dialog`? That seems to bit developers way... `more information dialog` might be more fitting in this context.

```suggestion
              "description": "Show an image card and corresponding entity states as an icon. The entities on the right side allow toggle actions, others show the more-info-dialog."
```